examples/sweeper/src/provider.ts

```diff
@@ -1,17 +1,18 @@
 import * as path from "path";
 import * as dotenv from "dotenv";
 import { ethers } from "ethers";
+
+// Load environment variables from `.env.local`
+dotenv.config({ path: path.resolve(process.cwd(), ".env.local") });
```

Comment:
nit: any reason this was moved up here instead of after all imports or after * imports?

Reply:
nope no particular reason... I guess the mentality was: external imports first, then env var stuff, then turnkey libraries, and then local imports (e.g. util files). just a matter of preference, nothing else